marklogic-data-hub-central/ui/src/components/sidebar/sidebar.module.scss

```diff
@@ -11,6 +11,10 @@
   padding: 0 0 0 5px;
 }
 
+.toggleIcon {
+    font-size: 14px;
```

Comment:
@AkshayBajajML, it looks like you were having an issue getting this CSS rule to take effect? That's why you had to use `const toggleIcon={ fontSize: '14px' };` below?

It can tricky getting your CSS styles to override what's being applied in a third-party component, but I think you can get this CSS rule to work with the following:

`font-size: 14px !important;`

(At least I see that working for me.)

Then you can 1) remove the `const toggleIcon={ fontSize: '14px' };`, 2) remove the `style={toggleIcon}` from the Icon component, and 3) add `className={styles.toggleIcon}` to the Icon component.